product/src/app.js

```diff
@@ -1,10 +1,12 @@
 import express from 'express';
-import { router } from './routes.js';
 import bodyParser from 'body-parser';
+import { productRouter } from './routes/productRoutes.js';
+import { userRouter } from './routes/userRoutes.js';
```

Comment:
a good practice is to create an index on the routes and make it responsible for receiving everything and just passing on a single router